maximbircu/devtools-library
# Add Bundler for iOS projects

Closes: #87

### What has been done
 1. Added `Gemfile` for `devtools/ios` and `samples/ios`. The `Gemfile.lock` is generated by Bundler.
 2. Set specific versions for iOS dependencies.

### How to test
1. Open `devtools/ios`. Run `bundle install && bundle exec pod install`. No errors should appear.
2. Open `samples/ios`. Run `bundle install && bundle exec pod install`. No errors should appear.
